Classes/Service/ContentService.php

```diff
@@ -225,8 +226,6 @@ public function moveRecord(array &$row, &$relativeTo, $parameters, DataHandler $
 			} elseif (0 < (integer) $relativeTo) {
 				// moving to first position in colPos, means that $relativeTo is the pid of the containing page
 				$row['sorting'] = $tceMain->getSortNumber('tt_content', 0, $relativeTo);
-				$row['tx_flux_parent'] = NULL;
```

Comment:
From my side it is correct.

There is a problem, when the element is moved with the arrow button to the first position. 
If  tx_flux_parent and tx_flux column are NULL, then the element will lose the connection to the gridcolumn and t will no longer appear on the page.